packages/map-template/src/components/MapTemplate/MapTemplate.jsx

```diff
@@ -326,6 +329,19 @@ function MapTemplate({ apiKey, gmApiKey, mapboxAccessToken, venue, locationId, p
         }
     }, [kioskOriginLocationId, mapsindoorsSDKAvailable]);
 
+    /*
+     * React on changes to the showKeyboard prop.
+     */
+    useEffect(() => {
+        if (mapsindoorsSDKAvailable) {
+            if (showKeyboard === 'true' || showKeyboard === true) {
```

Comment:
I think this should be handled where we handle query params and props (MapsIndoorsMap component). It should be converted to an actual boolean, so we don't have to make this guard in other components.

Reply:
Then we can also make this simpler, `setShowKeyboard(showKeyboard);`.

Reply:
Done ✅